### ISSUE TITLE
LSP: Refresh diagnostic on file deletion

### ISSUE THREAD

**Issue by hugocaillard** [hirosystems/clarinet#534]
Currently, the "problems" (warnings and errors) of a contract are still displayed in the "problems" panel when the file is deleted. It should be refreshed.

**Comment by lgalabru** [comment]
Are we still experiencing this issue @hugocaillard?
If so, do we want to keep it in our Q1 targets?

**Comment by hugocaillard** [comment]
Yes I'd like to keep this as part of a list of small improvements I can tackle in Q1 👍
